storage/excel/datasource.go

```diff
@@ -0,0 +1,207 @@
+package excel
+
+import (
+	"fmt"
+	"github.com/360EntSecGroup-Skylar/excelize"
+	"github.com/cube2222/octosql"
+	"github.com/cube2222/octosql/config"
```

Comment:
Please configure your editor to use goimports style formatting with grouping of standard library packages.

Like this:

```
import (
	"context"
	"strings"

	"github.com/cube2222/octosql"
	"github.com/cube2222/octosql/execution"
	"github.com/cube2222/octosql/execution/aggregates"
	"github.com/cube2222/octosql/physical/metadata"
	"github.com/pkg/errors"
)
```